plugins/filesystem/local/adapter/adapter.php

```diff
@@ -81,6 +84,98 @@ public function getFile($path = '/')
 	}
 
 	/**
+	 * Returns the folder or file information for the given path. The returned object
```

Comment:
There were some code syntax issues in this file, in incorrect format. I fixed them to Joomla! standard

there was like

`if(){

}` 

statements and changed them to

`if()
{

}`